tutorials/detecting-changes-in-sentinel-1-imagery-pt-4/Tutorial_pt_4.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<ul><li>"For this reason the method is not a particularly good fit to the GEE paradigm: Upscaling to very large geographical areas is not practical". This sentence gives me pause. By this I mean that people might find it confusing or unsure what it practically means. The confusion may come from people thinking "I thought the whole reason to use Earth Engine was to scale my analysis up" and it is unclear what the bounds of "very large geographical areas" is, and what can be done about it. In the context of this tutorial, it seems like the main limitation is with the interactive computation path that is used when making visualization requests i.e. the request is expected to finish in about 5 minutes otherwise it times out or crashes, which can happen when working on large areas at the native pixel resolution instead of relying on the aggregated tile layers. The large region issue is less of a concern when using the batch computation path (export the result to raster or vector). My suggestion is to say:</li></ul><blockquote>"For this reason it is best to limit your analysis to a region less than a few thousand square kilometers while interactively visualizing and exploring results using this notebook (for larger regions, [export the results](https://developers.google.com/earth-engine/guides/python_install#exporting-data)).</blockquote><ul><li>"<strong>Note:</strong>&nbsp;Depending on the position": you might style this note as italics</li><li>"As explained in Part 3 of this tutorial": hyperlink to the section you're referring to e.g.:</li></ul><p><code>[Part 3](https://developers.google.com/earth-engine/tutorials/community/detecting-changes-in-sentinel-1-imagery-pt-3#a_question_of_scale)</code></p><ul><li>"likelihood ratio test": hyperlink to the section e.g.</li></ul><p><code>[likelihood ratio test](https://developers.google.com/earth-engine/tutorials/community/detecting-changes-in-sentinel-1-imagery-pt-2#the_likelihood_ratio_test)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I see your point re caveat and I think your suggestion is excellent. Changed accordingly. The real limitation is the size of the intersection of the image footprints in the chosen time series. This is implied in the rest of the paragraph. </p>